libs/application/templates/reference-template/src/forms/ExampleForm.ts

```diff
@@ -40,53 +41,25 @@ export const ExampleForm: Form = buildForm({
       title: 'Table repeater',
       children: [
         buildTableRepeaterField({
-          id: 'rentalHousingLandlordInfoTable',
+          id: 'tableRepeaterField',
```

Comment:
@jonnigs might have something to say these days about changes to exampleForm until he gets his huge rework done hehe

Reply:
Yes, it would be nice to avoid merge conflicts if possible and not edit the reference-template

Reply:
Fair point about the merge conflict, I'll stomp this change.